client.go

```diff
@@ -60,6 +58,27 @@ var (
 	pathPattern    = regexp.MustCompile(`{[^}]+}`)
 )
 
+// Region specific to Recurly
+type Region string
+
+const (
+    US = Region("us")
+    EU = Region("eu")
+)
+
+// ClientOptions for a new API Client
+type ClientOptions struct {
+    region Region
```

Comment:
We will want to make this variable available outside the package, so it should be capitalized.

```suggestion
   // Region connects to the given data center geography
    Region Region
```